app/components/Tooltip/index.js

```diff
@@ -1,27 +1,194 @@
 import React from 'react';
-import H3 from 'components/H3';
+import PropTypes from 'prop-types';
+import groupBy from 'lodash/groupBy';
+import { FormattedMessage, injectIntl } from 'react-intl';
+import classNames from 'classnames';
+
+import isResourceAvailable from 'utils/isResourceAvailable';
+import getLocalizedString from 'utils/getLocalizedString';
 import VacancyLabel from 'components/VacancyLabel';
+import VacancyIcon from 'components/VacancyIcon';
 import CloseButton from 'components/CloseButton';
-import { TooltipWrapper, TooltipContainer } from './wrappers';
-
-const Tooltip = props => {
-  const showTooltip = props.visible;
-
-  if (showTooltip) {
-    return (
-      <TooltipContainer x={props.x} y={props.y}>
-        <TooltipWrapper className="animation-item">
-          <CloseButton tooltip onClick={props.onClick} />
-          <H3>{props.content.title}</H3>
-          {props.content.useRespa && (
-            <VacancyLabel variant="light" vacancy={props.content.available} />
-          )}
-        </TooltipWrapper>
-      </TooltipContainer>
-    );
+import categoryMessages from 'components/ButtonList/categoryMessages';
+import {
+  TooltipWrapper,
+  TooltipContainer,
+  Title,
+  Row,
+  RowLabel,
+} from './wrappers';
+import messages from './messages';
+import spaceTypeMessages from './spaceTypeMessages';
+
+function SubSpaceVacancyIcon({ availableCount, totalCount }) {
+  return (
+    <VacancyIcon
+      className={classNames({
+        available: availableCount === totalCount,
+        taken: availableCount === 0,
+        nonReservable: availableCount > 0 && availableCount < totalCount,
+      })}
```

Comment:
This is a bit confusing. You are checking if there is at least one room available but the field is named `nonReservable`. Should it be something like `partlyTaken` ?

Reply:
Yup, I agree. I have to refactor the label a bit in VAR-364. I didn't do it here in order to limit the scope of the changes (little good that did for me).

The root of the issue here is that the same hue is being used for "partly taken" and "not at all reservable", which I should resolve before I know what the correct refactor here is.